.github/workflows/tox.yml

```diff
@@ -1,17 +1,22 @@
 name: tox
-on: [push, pull_request]
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
 jobs:
   tox-jobs:
     strategy:
       fail-fast: false
       matrix:
-        job: [py37-flake83]  # ,pypy,pep8,py3pep8,setuppy,manifest]
-    runs-on: ubuntu-latest
+        os: [ubuntu-latest]  # [macos-latest, ubuntu-latest, windows-latest]
+        python-version: ['3.7', '3.9', '3.11']  # ,pypy3.9,pep8,py3pep8,setuppy,manifest]
```

Comment:
Did I miss 3.10?

Reply:
3.8 and 3.10… The idea was to run the odd versions and not the even versions to save resources but if you think we need to _cover the waterfront_, we can do that.

Reply:
Also, do we want to test on pypy3, run the manifest tests, etc?

Should we add pre-commit, black, mypy?  All optional.